tests/mocks.py

```diff
@@ -216,6 +286,9 @@ def user(self, *, username="imadueme_admin"):
         uri = "http://phabricator.test/p/{}".format(username)
         user = {
             "id": self._new_id(self._users),
+            "apiKey": "api-{}".format(
+                hashlib.md5(email.encode("utf-8")).hexdigest()[:12]
```

Comment:
super duper nit: I'd prefer to never use md5 for anything, even though in this case it's really not a security thing. Wanna switch this to `hashlib.sha256` like we use elsewhere?